### PR TITLE
Portfolio Project [ Pair Programming Project] for Microverse

### PR DESCRIPTION
**Pull request by mr-abdellah**
This is the final result of the whole Portfolio Project , and this is a Pair programming project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -160,6 +160,27 @@ <h4 class="card-paragraph">Skills</h4>
                 </div>
             </div>
         </section>
+        <section class="contact">
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <form class="my-form" action="https://formspree.io/f/xdobpeww" method="post">
+                <ul>
+                    <li><input type="text" name="fullname" id="username" maxlength="30" placeholder="Full Name" required></li>
+                    <li><input type="email" name="email" id="user_email" placeholder="Email Address" required></li>
+                    <li><textarea name="message" id="message" maxlength="500" cols="30" rows="10" required>Enter Text here</textarea></li>
```
Comment (AnselemOdims) [inline index.html:169]: - [ ] Please kindly include a placeholder text for the `textrea`

File: index.html [hunk 2]
```diff
@@ -160,6 +160,27 @@ <h4 class="card-paragraph">Skills</h4>
...
     </section>
+    <footer>
+        <ul class="social-media-links-footer">
+            <li><a href="https://twitter.com/belkaiduus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/Twitter.png" alt="Twitter Logo" ></a></li>
+            <li><a href="https://linkedin.com/in/belkaidus" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/Linkedin.png" alt="Linkedin Logo" ></a></li>
+            <li><a href="https://medium.com/" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/medium.png" alt="Medium Logo" ></a></li>
+            <li><a href="https://github.com/mr-abdellah" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/GitHub.png" alt="GitHub Logo"></a></li>
+            <li><a href="https://angel.co/" target="_blank" rel="noopener"><img rel="noreferrer" src="img/social/angellist.png" alt="Angellist Logo"></a></li>
+        </ul>
+        <h6>belkaid.abdulah@gmail.com</h6>
```
Comment (AnselemOdims) [inline index.html:183]: - [ ] Please kindly stick to the design guidelines(margins, padding, spacing etc) as much as possible. At the moment, your design does not match the original design

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @mr-abdellah , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by nicupop729 — APPROVED** [review]
#
Hi @mr-abdellah and @sajid-munawar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — APPROVED** [review]

**Comment by mr-abdellah** [comment]
hehe thank you bro @nicupop729 😄

**Pull request merged**